data/incipias/incipias.txt

```diff
@@ -0,0 +1,372 @@
+# Copyright (c) 2023 by RisingLeaf (https://github.com/RisingLeaf)
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Ship names:
+phrase "hicemus military"
+	word
+		"HIC "
+	phrase
+		"latin name"
+	word
+		"en #"
+	phrase
+		"hex color"
+
+phrase "incipias civilian"
+	phrase
+		"vocal consonant (capitalized)" 5
+		"consonant vocal (capitalized)" 10
+	word
+		"'"
+	phrase
+		"latin name"
+	word
+		"ius"
+		"ias"
+		"ium"
+		"iam"
+	word
+		" #"
+	phrase
+		"hex color"
+
+phrase "latin name"
+	phrase
+		"vocal consonant (capitalized)" 5
+		"consonant vocal (capitalized)" 10
+	phrase
+		"vocal consonant" 10
+		"consonant vocal" 5
+	phrase
+		"vocal consonant" 5
+		"consonant vocal" 10
+
+phrase "vocal consonant"
+	phrase
+		"vocal"
+	phrase
+		"consonant"
+
+phrase "consonant vocal"
+	phrase
+		"consonant"
+	phrase
+		"vocal"
+
+phrase "vocal consonant (capitalized)"
+	phrase
+		"vocal (capitalized)"
+	phrase
+		"consonant"
+
+phrase "consonant vocal (capitalized)"
+	phrase
+		"consonant (capitalized)"
+	phrase
+		"vocal"
+
+phrase "vocal"
+	word
+		"a"
+		"e"
+		"i"
+		"o"
+		"u"
+
+phrase "vocal (capitalized)"
+	word
+		"A"
+		"E"
+		"I"
+		"O"
+		"U"
+
+phrase "consonant"
+	word
+		"b"
+		"d"
+		"g"
+		"k"
+		"l"
+		"m"
+		"n"
+		"r"
+		"s"
+		"t"
+
+phrase "consonant (capitalized)"
+	word
+		"B"
+		"D"
+		"G"
+		"K"
+		"L"
+		"M"
+		"N"
+		"R"
+		"S"
+		"T"
+
+phrase "hex color"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+
+phrase "hex digit"
+	word
+		"0"
+		"1"
+		"2"
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+		"a"
```

Comment:
These should be capitalized as well, right?

Reply:
I normally do not capitalize my hex numbers?

Reply:
Okay. Resolve this.